src/WebJobs.Extensions.EasyTables/Bindings/EasyTableAttributeBindingProvider.cs

```diff
@@ -0,0 +1,85 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Globalization;
+using System.Reflection;
+using System.Threading.Tasks;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Azure.WebJobs.Host;
+using Microsoft.Azure.WebJobs.Host.Bindings;
+using Microsoft.WindowsAzure.MobileServices;
+using WebJobs.Mobile.EasyTables;
+
+namespace WebJobs.Extensions.EasyTables
+{
+    internal class EasyTableAttributeBindingProvider : IBindingProvider
+    {
+        private EasyTableConfiguration _easyTableConfig;
+        private JobHostConfiguration _jobHostConfig;
+        private INameResolver _nameResolver;
+
+        public EasyTableAttributeBindingProvider(JobHostConfiguration config, EasyTableConfiguration easyTableConfig, INameResolver nameResolver)
+        {
+            _jobHostConfig = config;
+            _easyTableConfig = easyTableConfig;
+            _nameResolver = nameResolver;
+        }
+
+        public Task<IBinding> TryCreateAsync(BindingProviderContext context)
+        {
+            if (context == null)
+            {
+                throw new ArgumentNullException(nameof(context));
+            }
+
+            ParameterInfo parameter = context.Parameter;
+            EasyTableAttribute attribute = parameter.GetCustomAttribute<EasyTableAttribute>(inherit: false);
+            if (attribute == null)
+            {
+                return Task.FromResult<IBinding>(null);
+            }
+
+            if (_easyTableConfig.MobileAppUri == null)
+            {
+                throw new InvalidOperationException(
+                    string.Format(CultureInfo.CurrentCulture,
+                    "The Easy Tables Uri must be set either via a '{0}' app setting or directly in code via EasyTableConfiguration.MobileAppUri.",
+                    EasyTableConfiguration.AzureWebJobsMobileAppUriName));
+            }
+
+            EasyTableContext easyTableContext = CreateContext(_easyTableConfig, attribute, _nameResolver);
+
+            IBindingProvider compositeProvider = new CompositeBindingProvider(new IBindingProvider[]
```

Comment:
I see Mike hasn't commented. I was expecting that now the helpers are in core you'd be able to follow the same pattern that his EventHub binding does, which doesn't use this array of composite binders. The bug I asked you to open above will track this - we need to get the right pattern in place for people ASAP before everyone starts copying this. These samples are basically templates for other community authors.